vnext/Microsoft.ReactNative.Managed/Properties/Microsoft.ReactNative.Managed.rd.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    This file contains Runtime Directives, specifications about types your application accesses
+    through reflection and other dynamic code patterns. Runtime Directives are used to control the
+    .NET Native optimizer and ensure that it does not remove code accessed by your library. If your
+    library does not do any reflection, then you generally do not need to edit this file. However,
+    if your library reflects over types, especially types passed to it or derived from its types,
+    then you should write Runtime Directives.
+
+    The most common use of reflection in libraries is to discover information about types passed
+    to the library. Runtime Directives have three ways to express requirements on types passed to
+    your library.
+
+    1.  Parameter, GenericParameter, TypeParameter, TypeEnumerableParameter
+        Use these directives to reflect over types passed as a parameter.
+
+    2.  SubTypes
+        Use a SubTypes directive to reflect over types derived from another type.
+
+    3.  AttributeImplies
+        Use an AttributeImplies directive to indicate that your library needs to reflect over
+        types or methods decorated with an attribute.
+
+    For more information on writing Runtime Directives for libraries, please visit https://go.microsoft.com/fwlink/?LinkID=391919
+-->
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+    <Library Name="Microsoft.ReactNative.Managed">
+
+        <!-- add directives for your library here -->
+        <Namespace Name="Microsoft.ReactNative.Managed" Dynamic="Required All" MarshalObject="Required All" />
```

Comment:
>Namespace [](start = 9, length = 9)

Nice! I did not know that we can associate the runtime data with the assembly. Doe sit mean that we can clean up the samples and new project template .rd.xml files?

Reply:
Haven't fully tried to publish to store. Okay to do in separate PR?

---
In reply to: [420386242](https://github.com/microsoft/react-native-windows/pull/4793#discussion_r420386242) [](ancestors = 420386242)

Reply:
We just need to see if the sample app continues to work in Release mode. It will crash if it does not work.

Reply:
Did we actually verify that the sample apps run? IIRC having a separate binary means the reflection will fail.